addons/survey/tests/test_survey.py

```diff
@@ -261,6 +261,66 @@ def test_partial_scores_simple_choice(self):
         self.assertEqual(user_input.scoring_percentage, 100)
         self.assertTrue(user_input.scoring_success)
 
+    def test_answer_time_limit_result(self):
```

Comment:
:socks: Should be defined after 'test_answer_display_name', to keep test ordering following the execution order (aka alphabetical)